src/CMakeLists.txt

```diff
@@ -97,6 +97,13 @@ add_subdirectory(app)
 
 set(DATA_OUTPUT_DIR ${CMAKE_BINARY_DIR}/bin/data)
 
+if(ENABLE_ASEPRITE_EXE)
+  set(main_target aseprite)
```

Comment:
Same here, probably we can just remove the `main_target` var (introduced in 11e4243d056d69e12f04bbb744ef53f0e4d46285 but not sure why, probably just for "searchability" purposes).